### ISSUE TITLE
gsl/benchmarks/blas_klee_bug.x86_64 spec is missing counter examples

### ISSUE THREAD

**Issue by delcypher** [delcypher/fp-benchmarks-imperial#1]
Our tool is reporting bugs in this benchmark but the 

```
/home/user/build_O2/benchmarks/c/imperial/gsl/benchmarks/blas_klee_bug.x86_64.bc (/home/dsl11/dev/klee-afr/experiments/first/wd/workdir-43/klee-wd)
Observed counter examples not listed in spec
  Test 000001 in /home/user/fp-bench/benchmarks/c/imperial/gsl/benchmarks/blas/blas-sym.c:50
  Test 000002 in /home/user/fp-bench/benchmarks/c/imperial/gsl/benchmarks/blas/blas-sym.c:49
```

but the spec doesn't list them as an expected counter examples.

They should be added.

@ccadar I think you're the author of this benchmark. Could you add them?

**Comment by ccadar** [comment]
So have we decided to always list the full list of counterexamples for each benchmark?

**Comment by delcypher** [comment]
@ccadar I would say it's preferable because it means a tool can't cheat (i.e. say it found a bug but gave a spurious source file location). Aachen (IIRC) have provided counter examples for all their benchmarks so we should too.

**Comment by delcypher** [comment]
Fixed by 2f968ab54f3938bb001bc3ac10fad41a616de713
